irene-perez/precurso/tema1/calculator_pro_Irene.js

```diff
@@ -0,0 +1,98 @@
+//Función para obtener los números que introduzca el usuario con prompt
+function getNumbers() {
+    //Variable con prompt donde el usuario introducirá los números. No se pone .slipt() aquí por si el usuario le da a cancelar, lo cual devuelve null y no se puede hacer este método sobre este valor 
+    let numbers = prompt("Introduzca los números, separados por comas, que desee");
+    //Ponemos un condicional si el usuario introduce algo que devuelve el valor null o no pone nada
+    if (!numbers || numbers === '') {
+        console.log("Por favor, asegúrese de introducir números, ¡inténtelo de nuevo! :)");
+        return getNumbers();
+    }
+    //Trim elimina espacios (si el usuario los pone entre la coma y el siguiente número) y se pone aquí el split porque aquí numbers ya sí tiene el valor de un string
+    numbers = numbers.trim().split(',');
+    //Se pone un condicional si el usuario añade un elemento pero no tiene valor numérico  
+    if (numbers.length === 1) {
+        const number = parseFloat(numbers[0]);
+        if (typeof number !== 'number' || isNaN(number)) {
+            console.log("Por favor, asegúrese de introducir números, ¡inténtelo de nuevo! :)");
+            return getNumbers();
+        }
+        return number;
+    }
+    //Se pone un bucle for para que vaya argumento por argumento validando que sean números
+    for (let i = 0; i < numbers.length; i++) {
```

Comment:
With such as simple for loops it is better for performance and readability to use for..in and for..of loops.

For example, for(let index of number){ ... }